pkg/authorization/admission/restrictusers/subjectchecker.go

```diff
@@ -18,7 +18,7 @@ import (
 // SubjectChecker determines whether rolebindings on a subject (user, group, or
 // service account) are allowed in a project.
 type SubjectChecker interface {
-	Allowed(kapi.ObjectReference, *RoleBindingRestrictionContext) (bool, error)
+	Allowed(rbac.Subject, *RoleBindingRestrictionContext) (bool, error)
```

Comment:
Maybe this should be `Allowed(subject *rbac.Subject, ctx *RoleBindingRestrictionContext) (allowed bool, err error)`

`*rbac.Subject` instead of `rbac.Subject`?

Reply:
makes it impossible to pass in nil, and basically just maintaining the previous interface, not a hard reason, but also saw no reason to change it, Subjects are pretty small